src/styles/nav.css

```diff
@@ -37,6 +32,106 @@
   background-color: var(--info);
 }
 
+/*new*/
+.top-nav .brand-nav {
+  background-color: var(--primary-color);
+  color: var(--txt-clr-secondary);
+  width: 100%;
+}
+.logo-brand {
+  font-size: var(--size-24);
+  color: var(--txt-clr-secondary);
+}
+.menu-icon {
+  display: none;
+}
+.bar1,
+.bar2,
+.bar3 {
+  width: 35px;
+  height: 5px;
+  background-color: var(--txt-clr-secondary);
+  margin: 6px 0;
+  transition: 0.4s;
+}
+.cross .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-9px, 6px);
```

Comment:
I don't think this is needed.

Reply:
thanks for pointing it out, I'll remove it